ItemRack/ItemRack.lua

```diff
@@ -424,10 +451,10 @@ function ItemRack.InitCore()
 	--if not disable_delayed_swaps then
 		-- in the event delayed swaps while casting don't work well,
 		-- make disable_delayed_swaps=1 at top of this file to disable it
-		-- ItemRackFrame:RegisterEvent("UNIT_SPELLCAST_START")
-		-- ItemRackFrame:RegisterEvent("UNIT_SPELLCAST_STOP")
-		-- ItemRackFrame:RegisterEvent("UNIT_SPELLCAST_SUCCEEDED")
-		-- ItemRackFrame:RegisterEvent("UNIT_SPELLCAST_INTERRUPTED")
+	ItemRackFrame:RegisterEvent("UNIT_SPELLCAST_START")
+	ItemRackFrame:RegisterEvent("UNIT_SPELLCAST_STOP")
+	ItemRackFrame:RegisterEvent("UNIT_SPELLCAST_SUCCEEDED")
+	ItemRackFrame:RegisterEvent("UNIT_SPELLCAST_INTERRUPTED")
```

Comment:
I disabled the events initially because I couldn't find documentation that they exist in classic, can you confirm that they are working as expected and exist in the game?

Reply:
Yes, these events work perfectly fine